float64 desLeftAPos
float64 desLeftBPos
float64 desLeftHipPos
float64 desRightAPos
float64 desRightBPos
float64 desRightHipPos

float64 leg_p_gain
float64 leg_d_gain
float64 hip_p_gain
float64 hip_d_gain

float64 legDuration
float64 hipDuration

